components/SubmissionComments.tsx

```diff
@@ -53,16 +106,54 @@ export const SubmissionComments: React.FC<{
               inline
             />
             {id === c.authorId ? (
-              <div>
-                <Button
-                  outline
-                  type="danger"
-                  color="danger"
-                  onClick={() => handleDeleteComment(c)}
-                >
-                  <TrashIcon />
-                </Button>
-              </div>
+              editing === i ? (
+                <div>
+                  <Button
+                    type="light"
+                    color="black"
+                    onClick={() => {
+                      handleDiscardChange()
+                    }}
+                    data-testid="discard-button"
+                  >
+                    Discard Changes
+                  </Button>
+                  <span> </span>
+                  <Button
+                    type="info"
+                    color="white"
+                    onClick={() => {
+                      handleUpdateComment(c, curInput)
+                    }}
+                    data-testid="save-button"
+                  >
+                    Save Changes
+                  </Button>
+                </div>
+              ) : (
+                <div>
+                  <Button
+                    type="info"
+                    size="sm"
+                    color="white"
+                    onClick={() => {
+                      handleStartEditing(i, c.content)
+                    }}
+                    data-testid="edit-button"
+                  >
+                    Edit
+                  </Button>
+                  <Button
+                    outline
+                    type="danger"
+                    color="danger"
+                    onClick={() => handleDeleteComment(c)}
+                    data-testid="delete-button"
```

Comment:
Maybe add a title="Delete comment" for better accessibility

Reply:
same as above, I dont think our Button component accepts a title prop

Reply:
Sounds good, we can maybe add this as a prop for the Button component in the future so we can get better accessibility.